lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -73,7 +81,7 @@ public static function get_theme_data( $deprecated = array(), $settings = array(
 		 * So we take theme supports, transform it to theme.json shape
 		 * and merge the static::$theme upon that.
 		 */
-		$theme_support_data = WP_Theme_JSON_Gutenberg::get_from_editor_settings( get_default_block_editor_settings() );
+		$theme_support_data = WP_Theme_JSON_Gutenberg::get_from_editor_settings( gutenberg_get_legacy_theme_supports_for_theme_json() );
```

Comment:
Other than this line, is any other change necessary?

Reply:
They are part of core. I will do that in another PR.